frame/authorship/src/lib.rs

```diff
@@ -194,50 +173,86 @@ decl_module! {
 				Self::prune_old_uncles(minimum_height)
 			}
 
-			<Self as Store>::DidSetUncles::put(false);
+			<DidSetUncles<T>>::put(false);
 
 			T::EventHandler::note_author(Self::author());
 
 			0
 		}
 
-		fn on_finalize() {
+		fn on_finalize(_: T::BlockNumber) {
 			// ensure we never go to trie with these values.
-			<Self as Store>::Author::kill();
-			<Self as Store>::DidSetUncles::kill();
+			<Author<T>>::kill();
+			<DidSetUncles<T>>::kill();
 		}
+	}
+
+	#[pallet::storage]
+	/// Uncles
+	pub(super) type Uncles<T: Config> = StorageValue<
+		_,
+		Vec<UncleEntryItem<T::BlockNumber, T::Hash, T::AccountId>>,
+		ValueQuery,
+	>;
+
+	#[pallet::storage]
+	/// Author of current block.
+	pub(super) type Author<T: Config> = StorageValue<_, Option<T::AccountId>, ValueQuery>;
```

Comment:
```suggestion
	pub(super) type Author<T: Config> = StorageValue<_, T::AccountId, OptionQuery>;
```

This solves the issue you had on line 255

Reply:
appreciate the help, that is what I initially suspected as well but that does not solve the issue you still need the some

Reply:
no with xlc change, you don't need the some anymore

Reply:
ah that was dumb lol, thanks :)